cli.py

```diff
@@ -1,190 +1,95 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-#
-# This is the first cli prototype for the cabinet core.
-#
-# References;
-#   argparse : https://docs.python.org/3/library/argparse.html
-#   cement   : http://builtoncement.com/2.10/index.html
-#
-# TODO:
-#   - Refactor the code to move each class, utils to its own module.
-
-import signal
-import argparse
-
-from sys import exit
-from cement.core.foundation import CementApp
-from cement.core.controller import CementBaseController, expose
+import click
+
+from utils import get_content_from_editor
 
 from cabinet_wrapper import CabinetWrapper
 
 
-VERSION = '0.0.1'
-BANNER = """
-Cabinet command line v{0}
-GNU GENERAL PUBLIC LICENSE
-""".format(VERSION)
```

Comment:
I removed this part unintentionally but afterwards I thought it was better not to have it since we are not versioning.

I'm not sure about how to manage versioning at this point, some other apps have their own `dev` version which are the nightly builds and I've seen them "versioning" with the short git hash from the commit that they are using.
I'd say that we are close enough to get a minimum working prototype to start versioning to wait before implementing something for this.
And, if we have some user complaining about something not working on some version before that happens, let's tackle that problem when it happens.
Does it sound good enough?

I noticed we haven't stated a license for this repo. I've added a `LICENSE` file, same as other repos.